[
  {
    "poster": {
      "poster_name": "Simone Neil-Jones",
      "poster_handle": "simoneneiljones"
    },
    "time_stamp": "2021-03-20 16:23:32",
    "text": "Hello everyone, when performing batch creates of contacts (API v3) I run into a problem. When I get an error of Category CONFLICT (so for example, if a contact I am creating has an email which already exists on HubSpot), I get an error message, but I don't get a list of successful operations as you do for other errors. I therefore have no way of knowing which operations in the batch worked and which didn't. This prevents me from using the Batch API in production. Does anyone else run into the same problem, or do you know someone I can report this potential bug to?"
  }
]